nativelib/src/main/resources/cre2.h

```diff
@@ -0,0 +1,188 @@
+/*
+  https://github.com/marcomaggi/cre2 (0.3.0)
+
+  Header  file  for  CRE2, a  C  language  wrapper  for RE2:  a  regular
+  expressions library by Google.
+
+  Copyright (c) 2012, 2015 Marco Maggi <marco.maggi-ipsu@poste.it>
+  Copyright (c) 2011 Keegan McAllister
+  All rights reserved.
+
+  For the license notice see the COPYING file.
+*/
+
+// Scala-Native version remove unused functions
```

Comment:
The same as in https://github.com/scala-native/scala-native/pull/588/files#r109922766